external/elemental.cmake

```diff
@@ -167,9 +190,26 @@ if(ENABLE_ELEMENTAL AND DEFINED ELEMENTAL_TAG)
     message (STATUS "** Done configuring Elemental")
   endif(error_code)
 
-  set(${ELEMENTAL_PACKAGE_NAME}_DIR ${ELEMENTAL_BINARY_DIR})
-  find_package(${ELEMENTAL_PACKAGE_NAME} REQUIRED
-               COMPONENTS REQUIRED ${ELEMENTAL_PACKAGE_NAME} pmrrr ElSuiteSparse)
+  execute_process(
```

Comment:
would file(MAKE_DIRECTORY ...) work here? https://cmake.org/cmake/help/v3.2/command/file.html 
p.s. also see instances of invoking cmake -E make_directory above